app/controllers/pages_controller.rb

```diff
@@ -1,10 +1,28 @@
 # frozen_string_literal: true
 
 class PagesController < ApplicationController
+  before_action :set_date, only: [:about_ambler]
+  
   def home
   end
 
   def get_highlights
     @highlights = Highlight.where(promoted: true).take(4)
   end
+
+  def ambler
+  end
+
+  def index
```

Comment:
I'm not sure why this is here?

Reply:
In my limited experience, I need a def in the controller in order for the route to work.  This may be untrue.